xopt/bayesian/utils.py

```diff
@@ -203,11 +202,13 @@ def sampler_evaluate(inputs, evaluate_f, *eval_args, verbose=False):
         outputs = evaluate_f(inputs, *eval_args)
 
     except Exception as ex:
+        # No need to print a nasty exception
+        logger.warning(f'Exception caught in {__name__}')          
```

Comment:
Warning seems weird here with a message talking about `Exception`.
If this is an exception consider using `logger.exception` or `logger.error` if you don't want the exception stacktrace to be displayed.

Reply:
The problem is that we have `xopt.output_notebook()` that redirects logger messages to stdout, and `logger.exception` starts to clutter that.

Reply:
In this case, logger.error will be better unless this is really just a warning and safe to be ignored

Reply:
Changed to `logger.error`